### PR TITLE
DOC: Add/remove spaces in snippets and re-format here and there

### PR DESCRIPTION
**Pull request by rinarakaki**

### CODE REVIEW

File: doc/source/user/quickstart.rst [hunk 1]
```diff
@@ -185,27 +185,27 @@ state of the memory. By default, the dtype of the created array is
...
     <BLANKLINE>
            [[1, 1, 1, 1],
             [1, 1, 1, 1],
             [1, 1, 1, 1]]], dtype=int16)
-    >>> np.empty( (2,3) )                                 # uninitialized
-    array([[  3.73603959e-262,   6.02658058e-154,   6.55490914e-260],  # may vary
-           [  5.30498948e-313,   3.14673309e-307,   1.00000000e+000]])
+    >>> np.empty((2, 3))  # uninitialized, the result may vary
+    array([[3.73603959e-262, 6.02658058e-154, 6.55490914e-260],
+           [5.30498948e-313, 3.14673309e-307, 1.00000000e+000]])
```
Comment (eric-wieser) [inline doc/source/user/quickstart.rst:198]: "may vary" is a special marker understood by our CI tools, I think you might need to put it back where it was.

File: doc/source/user/quickstart.rst [hunk 2]
```diff
@@ -280,8 +280,8 @@ central part of the array and only prints the corners::
 
     >>> print(np.arange(10000))
     [   0    1    2 ... 9997 9998 9999]
-    >>>
-    >>> print(np.arange(10000).reshape(100,100))
+    >>>
```
Comment (eric-wieser) [inline doc/source/user/quickstart.rst:283]:
```suggestion
    >>>
```
otherwise someone's editor will just trim this again.

File: doc/source/user/quickstart.rst [hunk 3]
```diff
@@ -308,35 +308,35 @@ created and filled with the result.
...
+    >>> a = np.array([20, 30, 40, 50])
+    >>> b = np.arange(4)
     >>> b
     array([0, 1, 2, 3])
-    >>> c = a-b
+    >>> c = a - b
     >>> c
     array([20, 29, 38, 47])
-    >>> b**2
+    >>> b ** 2
```
Comment (eric-wieser) [inline doc/source/user/quickstart.rst:318]: I'd be inclined to avoid spaces around `**`

**Review by eric-wieser — COMMENTED** [review]
This largely looks great, thanks!

My only objections would be subjective, that:

* Putting a trailing space after `>>>` isn't helpful, it will probably be stripped in a later commit by accident
* Spaces around `b**2` as `b ** 2` etc seems unusual
* I think some of the column alignment of comments was deliberate, but I don't feel strongly

**Comment by charris** [comment]
@rnarkk Could you make the suggested fixes?

File: doc/source/user/quickstart.rst (hunk 1 above)
Comment (mattip) [reply doc/source/user/quickstart.rst:198]: Interesting that this did not fail.

**Comment by mattip** [comment]
Thanks @rnarkk

**Comment by rnarkk** [comment]
HI,

Thank you so much for providing helpful reviews.
Besides the commit I newly added, I would leave memo on the reviews:

- The special marker '# may very' seemed not to be needed as all the CI's checks have passed.
- I added a space every after '>>>' for Sphinx to recognise them as prompts. The concern - someone's editor would remove them by chance - should be cared by man/machine's review or finding Sphinx's settings.
- Basically I put 2 spaces between code and comments but some comments are vertically aligned so that they look a comparison. Yet I'm not sure if I've done it all correctly nor there was sort of 'inter-snippet' alignment. If someone would tell me the basic rules, I'll follow them next time I submit PR.

Thank you!

File: doc/source/user/quickstart.rst (hunk 1 above)
Comment (eric-wieser) [reply doc/source/user/quickstart.rst:198]: This is failing in other PRs now, eg https://app.circleci.com/pipelines/github/numpy/numpy/5410/workflows/93d901be-60df-4a79-9d56-a35bdcc5b2f2/jobs/17498/parallel-runs/0/steps/0-106

**Comment by eric-wieser** [comment]
CI only passed because for some reason circle-ci did not run at all on this PR.

File: doc/source/user/quickstart.rst (hunk 1 above)
Comment (mattip) [reply doc/source/user/quickstart.rst:198]: potential fix in gh-18008
